examples/README.md

```diff
@@ -366,13 +366,12 @@ cargo install wasm-bindgen-cli
 
 ### Build & Run
 
-Following is an example for `headless_wasm`. For other examples in wasm/ directory,
-change the `headless_wasm` in the following commands **and edit** `examples/wasm/index.html`
-to point to the correct `.js` file.
+Following is an example for `lighting`. For other examples, change the `lighting` in the
+following commands.
 
 ```sh
-cargo build --example headless_wasm --target wasm32-unknown-unknown --no-default-features
-wasm-bindgen --out-dir examples/wasm/target --target web target/wasm32-unknown-unknown/debug/examples/headless_wasm.wasm
+cargo build --release --example lighting --target wasm32-unknown-unknown
+wasm-bindgen --out-name wasm_example --out-dir examples/wasm/target --target web target/wasm32-unknown-unknown/release/examples/lighting.wasm
 ```
```

Comment:
Just because running a Wasm build does still require some more setup compared to native targets, I would like to add a short explanation here. Something along the lines of: The first command creates a Wasm binary. With the second command, we let `wasm-bingen` generate some javascript glue code, that enables us to run the Wasm binary from an `index.html` file (link to the file here).